lib/backend/buffer.go

```diff
@@ -319,6 +344,9 @@ func (w *BufferWatcher) String() string {
 
 // Events returns events channel
 func (w *BufferWatcher) Events() <-chan Event {
+	w.bmu.Lock()
+	defer w.bmu.Unlock()
+	w.flushBacklog()
```

Comment:
Could you add a comment mentioning why it's OK not to check the result of flushBacklog here (and thus miss a possible backlogSince update)?